ibis_substrait/tests/compiler/snapshots/test_tpch/test_compile/tpc_h111/tpc_h111.json

```diff
@@ -0,0 +1,515 @@
+                                  }
+                                }
+                              }
+                            ]
+                          }
+                        }
+                      }
+                    },
+                    "groupings": [
+                      {
+                        "groupingExpressions": [
+                          {
+                            "selection": {
+                              "directReference": {
+                                "structField": {}
+                              },
+                              "rootReference": {}
+                            }
+                          }
+                        ]
+                      }
+                    ],
+                    "measures": [
+                      {
+                        "measure": {
+                          "functionReference": 2,
+                          "phase": "AGGREGATION_PHASE_INITIAL_TO_RESULT",
+                          "outputType": {
+                            "decimal": {
+                              "scale": 2,
+                              "precision": 38,
+                              "nullability": "NULLABILITY_NULLABLE"
+                            }
+                          },
+                          "arguments": [
+                            {
+                              "value": {
+                                "scalarFunction": {
+                                  "functionReference": 3,
+                                  "outputType": {
+                                    "decimal": {
+                                      "scale": 4,
+                                      "precision": 30,
+                                      "nullability": "NULLABILITY_NULLABLE"
+                                    }
+                                  },
+                                  "arguments": [
+                                    {
+                                      "value": {
+                                        "selection": {
+                                          "directReference": {
+                                            "structField": {
+                                              "field": 3
+                                            }
+                                          },
+                                          "rootReference": {}
+                                        }
+                                      }
+                                    },
+                                    {
+                                      "value": {
+                                        "cast": {
+                                          "type": {
+                                            "decimal": {
+                                              "scale": 2,
+                                              "precision": 15,
+                                              "nullability": "NULLABILITY_NULLABLE"
```

Comment:
This line is the only diff relative to `tpc_h110.json`: 

<img width="1512" alt="Screenshot 2023-12-30 at 16 13 06" src="https://github.com/ibis-project/ibis-substrait/assets/7870972/484c566f-ccbe-4308-bd97-10cc7c0a2152">

This changed because join outputs are now nullable because of [this change](https://github.com/ibis-project/ibis/pull/7596/files#diff-63b3fef44a98e55148586e372709a1c87c52dc23d3053db8654847306927055bR288).